src/components/ebay-bar-chart/component.js

```diff
@@ -0,0 +1,346 @@
+                        }
+                    });
+                });
+            } else {
+                // setup html for single series tooltip
+                // add the label value from the data point to the tooltip below the date
+                s += `<div><span>${this.point.label}</span></div>`;
+            }
+            return s;
+        };
+    }
+    tooltipPositioner(labelWidth, labelHeight) {
+        const chartPosition = this.chart.pointer.getChartPosition(); // returns the pointers top and left positions
+        const hpIndex = this.chart.hoverPoint.index; // reference to the index of the original hovered point of the series
+        const hoverPoint = this.chart.hoverPoint, // reference to the original hovered point of the series
+            y = // setting the y position of the tooltip to the top of the hovered stack of points
+                chartPosition.top +
+                hoverPoint.series.yAxis.top +
+                this.chart.series[this.chart.series.length - 1].data[hpIndex].shapeY -
+                labelHeight -
+                15; // adjust for the arrow
+        let x = // setting the x position of the tooltip based on the center of the hovered stack of points
+            chartPosition.left +
+            hoverPoint.dlBox.x +
+            hoverPoint.dlBox.width * 0.5 -
+            labelWidth * 0.5 +
+            3; // offset padding
+        // check left bound and adjust if the tooltip would be clipped
+        if (x < 6) {
+            x = 6;
+        }
+        // check right bound and adjust if the tooltip would be clipped
+        if (x + hoverPoint.dlBox.width > chartPosition.left + this.chart.chartWidth - 6) {
+            x = chartPosition.left + this.chart.chartWidth - hoverPoint.dlBox.width - 6;
+        }
+        return { x, y }; // return the tooltip x and y position
+    }
+
+    getTooltipConfig() {
+        return {
+            formatter: this.tooltipFormatter(),
+            useHTML: true, // allows defining html to format tooltip content
+            backgroundColor: tooltipBackgroundColor, // sets tooltip background color
+            borderWidth: 0, // hide the default border stroke
+            borderRadius: 10, // set the border radius of the tooltip
+            outside: true, // used to render the tooltip outside of the main SVG element
+            shadow: false, // hide the default shadow as it conflicts with designs
+            style: {
+                filter: tooltipShadows, // sets tooltip shadows
+                fontSize: '12px',
+            },
+            // this callback function is used to position the tooltip at the top of the stacked bars
+            positioner: this.input.stacked && this.tooltipPositioner,
+        };
+    }
+
+    legendItemClick() {
+        // returns a function so that can access input values
+        const component = this; // component reference used in item click functions that don't have the same scope
```

Comment:
This isn't even needed. We can simply just add a `const stacked = this.input.stacked` at the top.
If we are worrying about change in inputs, we might need to revisit how to initialize highcharts.